internal/images/custom-scorecard-tests/rbac/scorecard_role.yaml

```diff
@@ -150,3 +150,12 @@ rules:
   - namespaces
   verbs:
   - create
+  - delete
+- apiGroups:
+  - operator.cryostat.io
+  resources:
+  - cryostats
+  verbs:
+  - create
+  - delete
+  - get
```

Comment:
There is already a rule to create Cryostat above?

Reply:
not in the ClusterRole

Reply:
Otherwise I can't create a Cryostat resource in the targetNamespace

Reply:
Ahh okay thanks! I missed that haha :))